### ISSUE TITLE
Blank screen with ERROR message when loading project in tW

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#5980]
1.1.3 (f505bc4) 
https://git.door43.org/qa99/en_prct_tit_book
Import this project and load it in tW
![image](https://user-images.githubusercontent.com/8171759/56145322-aff84900-5f69-11e9-8036-99bda2857178.png)

This project loaded properly in tW in 1.0.2 and 1.1.2. When upgrading to 1.1.3 with the project previously imported, the migration seems to take a long time and logs errors. The migrated projects will not properly load in tW either.

[Mon Apr 15 2019.log.zip](https://github.com/unfoldingWord-dev/translationCore/files/3080956/Mon.Apr.15.2019.log.zip)

**Comment by PhotoNomad0** [comment]
Verified in https://tc-builds.door43.org/build/release-v1.1.3/translationCore-macos-x64-1.1.3-cf7905c.dmg

**Comment by cckozie** [comment]
1.1.3 (cf7905c)
The problem is fixed when I start with 1.1.3 and import the designated project.
When I start with the project already imported into 1.1.2 and upgraded to 1.1.3, the first two times I attempted this the program crashed about the time I was expecting the loading dialog to disappear (about 3 minutes). The first time this happened, the program worked fine when I restarted it. The second time I saw this failure I had to restart tC twice in order to get it to run.
At that point I rebooted my Mac and this time the app opened correctly after the upgrade to 1.1.3. I repeated the process once more and it worked correctly.
The console would never open before the crash. Here is the crash report from the OS:
[crash.txt.zip](https://github.com/unfoldingWord-dev/translationCore/files/3090836/crash.txt.zip)
